Assembus/Assets/Scripts/Services/ConfigurationManager.cs

```diff
@@ -59,16 +59,17 @@ private void LoadConfig()
         ///     Writes ConfigFileStruct FileStruct to XML file.
         ///     If writing fails, return false and write to debug log
         /// </summary>
-        public void SaveConfig()
+        /// <returns>True if the saving was successful</returns>
+        public bool SaveConfig()
         {
             try
             {
```

Comment:
Warum wurde die BEschreibung gelöscht, bzw wo ist die Beschreibung ?

Reply:
The description is still there I only added the returns description